doc/specific_iocs/dae/datastreaming/Datastreaming---Sample-Environment.md

```diff
@@ -2,16 +2,25 @@
 
 All IBEX instruments are currently forwarding their sample environment PVs into Kafka. This is done in two parts:
 
-### BlockserverToKafka
+## BlockserverToKafka
+
 This is a Python process that runs on each NDX (see code [here](https://github.com/ISISComputingGroup/EPICS-inst_servers/tree/master/BlockServerToKafka)) it monitors the blockserver config PVs and any time the config changes it pushes a new configuration to the forwarder, via a Kafka topic `forwarder_config`. This is a process written and managed by IBEX developers.
 
-The instrument name for the BlockServerToKafka service is `BSKAFKA`. 
+The `procserv` name for the BlockServerToKafka service is `BSKAFKA`. 
 
-### Forwarder
-This is a Python program responsible for taking the EPICS data and pushing into Kafka. ISIS currently has two instances of the forwarder running (one for the production and one for development). They are both running as services (Developer Forwarder and Production Forwarder) under `nssm` on NDADATASTREAM, which can be accessed via the `ibexbuilder` account. The configuration files and logs for these forwarders are located in `C:\Forwarder\dev_forwarder` and `C:\Forwarder\prod_forwarder`. The actual source lives in `C:\forwarder\fw_py`, updating it is a case of running `git pull` and re-installing the requirements. 
+## Forwarder
 
 Source for the forwarder is available [here](https://github.com/ess-dmsc/forwarder)
 
-We don't currently run this, and need to figure out topology ie. running a central forwarder, one per instrument and so on.
+We don't currently run this for every instrument, and need to figure out topology ie. running a central forwarder, one per instrument and so on.
+
+### Forwarder on HIFI
+
+HIFI has an instance of the forwarder currently running under procserv within IBEX for the SuperMuSR project.
+
+in `C:\Instrument\Apps\EPICS\utils\build_ioc_startups.py` we have hotfixed this line: 
+`ioc_startups.add("FWDR", IocStartup(os.path.join("C:\\", "instrument", "dev", "forwarder"), description="forward epics to kafka", exe="forwarder_launch.bat", iocexe="procServ.exe"))`
+
+to add a Procserv entry that runs it. 
 
-_NB: The forwarder was previously written in C++ but has now migrated to Python instead._
\ No newline at end of file
+HIFI also has a modified `ISIS/inst_servers/master/start_bs_to_kafka_cmd.bat` which points to the SuperMuSR Redpanda instance. 
```

Comment:
Modified by what process - do we have to reapply the modification after every release?